redhat-cop/containers-quickstarts
# Add support for pushing ubi8 git image to Quay

#### What is this PR About?
Add support for pushing ubi8 git image to Quay, fixes #322 

If we're happy with this workflow, I'll also submit an update to https://redhat-cop.github.io/ci/publishing-images.html describing how to use this workflow

#### How do we test this?
CI build should run when anything in `gitlab-runners/ubi8-git` directory is updated.
An image should get published to `quay.io/redhat-cop/gitlab-runners-ubi8-git` when `gitlab-runners/ubi8-git/version.json` is updated and merged to `master`. The image will be tagged with the version in the `version.json` file + `latest`.

cc: @redhat-cop/day-in-the-life
